net-im/tdesktop/patches/tdesktop-1.1.23-source2.patchset

```diff
@@ -0,0 +1,22 @@
+From 087620cc3cd0cc4eb1a7a250a8b35ced09791f17 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Zolt=C3=A1n=20Mizsei?= <zmizsei@extrowerk.com>
+Date: Sun, 10 Sep 2017 20:40:55 +0200
+Subject: Enable threading on Haiku
+
+
+diff --git a/libtgvoip-757a5d8ec31aadcf387ac3cb135256152c18de1c/threading.h b/libtgvoip-757a5d8ec31aadcf387ac3cb135256152c18de1c/threading.h
+index 8a6f438..d12544f 100644
+--- a/libtgvoip-757a5d8ec31aadcf387ac3cb135256152c18de1c/threading.h
++++ b/libtgvoip-757a5d8ec31aadcf387ac3cb135256152c18de1c/threading.h
+@@ -7,7 +7,7 @@
+ #ifndef __THREADING_H
+ #define __THREADING_H
+ 
+-#if defined(_POSIX_THREADS) || defined(_POSIX_VERSION) || defined(__unix__) || defined(__unix) || (defined(__APPLE__) && defined(__MACH__))
++#if defined(_POSIX_THREADS) || defined(_POSIX_VERSION) || defined(__unix__) || defined(__unix) || defined(__HAIKU__) || (defined(__APPLE__) && defined(__MACH__))
```

Comment:
This shouldn't be needed, because we define _POSIX_THREADS...
https://github.com/haiku/haiku/blob/master/headers/posix/unistd.h#L44

Reply:
Maybe just include unistd.h before checking for _POSIX_THREADS

Reply:
Thanks, i'll check it.